weave/trace/op.py

```diff
@@ -107,6 +110,7 @@ class Op(Protocol):
     """
 
     name: str
+    display_name: str
```

Comment:
I think we are missing something a bit critical here: serialization and deserialization.  (maybe this is something we want to acknowledge is a gap in the spirit of MVP?). But when you "get" an op, the custom display name function is not coming long for the ride.

Reply:
Yep, ack this is a gap.  We'll have to think about how to handle this.

Reply:
can we add code note to this effect?